2021/37xxx/CVE-2021-37139.json

```diff
@@ -1,18 +1,78 @@
 {
```

Comment:
CNA(s) of user (JFROG) do not match owning CNA.

Reply:
This was a legacy CVE ID, changed to CVE-2021-43305